apps/web/components/AppListCard.tsx

```diff
@@ -63,31 +63,27 @@ export default function AppListCard(props: AppListCardProps) {
   }, []);
 
   return (
-    <div className={`${highlight ? "bg-yellow-100" : ""}`}>
-      <div className="flex gap-x-3 p-4">
-        {logo ? <img className="h-10 w-10" src={logo} alt={`${title} logo`} /> : null}
-        <div className="flex grow flex-col gap-y-1 truncate">
-          <div className="flex items-center gap-x-2">
-            <h3 className="truncate text-sm font-semibold text-gray-900">{title}</h3>
-            <div className="flex items-center gap-x-2">
-              {isDefault && <Badge variant="green">{t("default")}</Badge>}
-              {isTemplate && <Badge variant="red">Template</Badge>}
-            </div>
+    <ListItem
+      heading={title}
+      subHeading={description}
+      leftNode={logo ? <img className="h-10 w-10" src={logo} alt={`${title} logo`} /> : null}
+      badgePosition="heading"
+      badges={
```

Comment:
Personally I'm a big proponent if compound components, because that gives you even more flexibility in how this component will render, without needing to add a lot of props (and possible prop combinations). 

There's no need to refactor everything now, since most of our code bases uses the approach you use. With this comment I mainly want to plant this idea in your head, since in my opinion it makes the code so much more flexible, and your components so much more simple 😁

It would look something like: 

```
<ListItem>
   <ListItem.Visual><img src={logo} /></ListItem.Visual>
   <ListIem.Content>
    <ListItem.Heading>
      {title}
      <ListItem.Badges>
        {isDefault && <Badge variant="green">{t("default")}</Badge>}
         {isTemplate && <Badge variant="red">Template</Badge>}
      </ListIem.Badges>
    </ListItem.Heading>
    <ListItem.SubHeading>{description}</ListItem.SubHeading>
  </ListItem.Content>
  <ListItem.Actions>{actions}</ListItem.Actions>
</ListItem>
```